db/error_handler.cc

```diff
@@ -761,22 +758,12 @@ void ErrorHandler::RecoverFromRetryableBGIOError() {
         // recover from the retryable IO error and no other BG errors. Clean
         // the bg_error and notify user.
         TEST_SYNC_POINT("RecoverFromRetryableBGIOError:RecoverSuccess");
-        Status old_bg_error = bg_error_;
-        is_db_stopped_.store(false, std::memory_order_release);
```

Comment:
May need to move `is_db_stopped_` to ClearBGError() too.

Reply:
Yeah, I guess this wasn't supposed to be missing in `ClearBGError()` in the first place. Thanks for catching this.